.github/workflows/build-push.yml

```diff
@@ -0,0 +1,28 @@
+name: Build
+
+on:
+  push:
+    branches: [ main ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.17.8
+
+      - name: Build
+        run: go build -v ./...
+
+      - name: Set up Docker
+        uses: docker/setup-buildx-action@v1
+
+      - name: Build Docker images
+        run: docker build -t obscuro_enclave - < dockerfiles/enclave
+
+      - name: Test
+        run: go test -v ./...
```

Comment:
github doesn't seem to like there no newline here

Reply:
Thanks, will fix.